src/runtime/renderscript_dev.ll

```diff
@@ -0,0 +1,29 @@
+target datalayout = "e-m:e-p:32:32-i64:64-v128:64:128-a:0:32-n32-S64"
```

Comment:
Yeah, that's what I was wondering about. Surprising that it expects an arm target triple for GPU code though.